src/main/scripts/src/app/core/rest/collection.service.ts

```diff
@@ -113,9 +113,20 @@ export class CollectionService {
       .catch(CollectionService.handleGlobalError);
   }
 
+  private toDto(collection: Collection): Collection {
```

Comment:
Is this method really needed? It just creates the object with the same structure as the already existing one. Or where is the catch?

Reply:
The collection it receives from backend has additional attributes (like the `lastTimeUsed`) that caused the update method to fail.